packages/hub/services/scheduled-payments/fetcher.ts

```diff
@@ -61,8 +72,8 @@ export default class ScheduledPaymentsFetcherService {
               )
             ) 
             OR (
-              scheduled_payments.recurring_day_of_month > 0 
-              AND scheduled_payments.recurring_until >= ${_nowUtc}
+              scheduled_payments.recurring_day_of_month > 0
+              AND scheduled_payments.recurring_until >= ${nowString}::timestamp
```

Comment:
To ensure UTC. We could use `now() at time zone 'utc'` but we want to have the possibility to mock current utc time for testing (using the clock service stub)